rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/rest/TestExecutionLogService.xtend

```diff
@@ -0,0 +1,110 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.rcp4.views.tcltestrun.rest
+
+import java.nio.file.Files
+import javax.json.Json
+import javax.ws.rs.GET
+import javax.ws.rs.Path
+import javax.ws.rs.PathParam
+import javax.ws.rs.Produces
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.Response
+import org.eclipse.xtend.lib.annotations.Accessors
+import org.testeditor.rcp4.views.tcltestrun.model.TestExecutionManager
+import javax.json.JsonObjectBuilder
+import java.util.List
+import org.testeditor.rcp4.views.tcltestrun.model.TestLogGroupBuilder
+import org.testeditor.rcp4.views.tcltestrun.model.TestLogGroup
+import org.testeditor.rcp4.views.tcltestrun.model.TestLogGroupComposite
+import javax.json.JsonArray
+
+@Path("/testexeclogs")
+class TestExecutionLogService {
+
+	@Accessors
+	TestExecutionManager testExecutionManager
+
+	@GET
+	@Produces(MediaType::APPLICATION_JSON)
+	def Response getTestLogExeutionsList() {
+		val result = Json.createObjectBuilder
```

Comment:
I disagree with embedding a separate library here. Instead Gson should be used which is already in the target platform.